### PR TITLE
glibc 2.23 and 2.24 upgrade

### PR DESCRIPTION
**Pull request by sknsean**

### CODE REVIEW

File: recipes/glibc/glibc.inc [hunk 1]
```diff
@@ -40,6 +40,7 @@ MIN_KERNEL_VERSION[expand] = "3"
 do_configure[prefuncs] += "do_configure_siteinfo"
 do_configure_siteinfo() {
 	echo "libc_cv_ssp=no" > config.cache
+	echo "libc_cv_ssp_strong=no" >> config.cache
```
Comment (esben) [inline recipes/glibc/glibc.inc]: Is this still needed now that we have merged #109?

Comment (sknsean) [inline recipes/glibc/glibc.inc]: I guess not...

**Comment by esben** [comment]
LGTM, but waiting for CI status.

**Comment by sknsean** [comment]
Hmm this seems to be okay, but the powerpc builder is showing an error...
http://buildbot.oe-lite.org/builders/machine-powerpc-e300c3-glibc/builds/771/steps/bake%20world/logs/stdio
The commit ID the fetcher is trying is 3a188eb4e641d2df0cfd352fd09232347f28fbe1 and from the log it fetches:  f23d7d2..3a188eb  release/2.23/master -> release/2.23/master

**Comment by esben** [comment]
Hmm... Maybe there was a force push upstream, and we need to clean out our cached ingredients.

**Comment by sknsean** [comment]
This depends on #143

**Comment by esben** [comment]
@sknsean could you rebase this to current master?

**Comment by kimrhh-prevas** [comment]
before doing that we should probably try to fix the dmalloc build issue that @villemoes have been trying to fi

**Pull request merged**